### ISSUE TITLE
[5.2] Accessibility issues in pagebreak plugin

### ISSUE THREAD

**Issue by drmenzelit** [joomla/joomla-cms#44429]
The issue is probably related with Bootstrap changes that were not adopted in the Joomla code

### Steps to reproduce the issue
Create an article with some page breaks. Set the plugin to display as slider.
Use some accessibility tool like axe DevTools to check the site

### Expected result
Correct code, no accessibility issues

### Actual result
Accessibility issues
![Image](https://github.com/user-attachments/assets/75a5ddf1-b081-42b6-953c-b89b3004d9a4)

![Image](https://github.com/user-attachments/assets/e797c366-643e-409d-9507-9d808fb81e8e)

![Image](https://github.com/user-attachments/assets/c302a05b-3899-4a6b-b5d9-244c7939064f)

### Additional comments

**Comment by dgrammatiko** [comment]
Just use `<detail name="something">` for accordions: https://caniuse.com/mdn-html_elements_details_name

- No JS needed
- Pure HTML
- Totally accessible

**Comment by drmenzelit** [comment]
@dgrammatiko do you want to rewrite the Pagebreak plugin and the used accordion function from Joomla?

**Comment by drmenzelit** [comment]
@Quy thanks, I had totally forgotten that PR :-)

**Comment by dgrammatiko** [comment]
I don't have the time to do so but should be straight forward with layouts/sublayouts

**Comment by drmenzelit** [comment]
Closing as we already have a PR #40578
